_posts/2020-04-08-our-rails-upgrade-process.markdown

```diff
@@ -3,8 +3,11 @@ layout: post
 title: "Our Rails Upgrade Process: How to bundle update rails"
 date: 2020-04-08 15:00:00
 reviewed: 2020-04-08 15:00:00
-categories: ["rails", "upgrade"]
+categories: ["rails", "upgrades"]
```

Comment:
@cleicar I think you want to have both `upgrade` and `upgrades`, right?

Reply:
@lubc no, because if we put both the url will be 
https://www.fastruby.io/blog/rails/upgrade/upgrades/our-rails-upgrade-process.html